src/app/app.component.ts

```diff
@@ -36,6 +36,10 @@ import { FeaturesService } from './services/features.service';
 import { SharedService } from './services/shared.service';
 import { ThemingService } from './services/theming.service';
 import { FooterLink } from './shared/footer/footer.component';
+import { ModuleRegistry } from '@ag-grid-community/core';
+import { ClientSideRowModelModule } from '@ag-grid-community/client-side-row-model';
+
+ModuleRegistry.registerModules([ClientSideRowModelModule]);
```

Comment:
This seems relevant to getting agGrid to initialise, but is there a better place to put this? For example, since we so far only use agGrid for bulk editing, could this be set in the `BulkEditModule`? It would then be closer to where it is used, at the moment someone might have trouble tracking down why this line of code is here in the app root itself.